packages/gatsby-cli/src/init-starter.ts

```diff
@@ -117,13 +119,17 @@ const install = async rootPath => {
   }
 }
 
-const ignored = path => !/^\.(git|hg)$/.test(sysPath.basename(path))
+const ignored = (path: string): boolean =>
+  !/^\.(git|hg)$/.test(sysPath.basename(path))
 
 // Copy starter from file system.
-const copy = async (starterPath: string, rootPath: string) => {
+const copy = async (
+  starterPath: string,
+  rootPath: string
+): Promise<boolean> => {
   // Chmod with 755.
   // 493 = parseInt('755', 8)
-  await fs.mkdirp(rootPath, { mode: 493 })
+  await fs.ensureDir(rootPath, { mode: 493 })
```

Comment:
[`ensureDir` is an alias of `mkdirp`](https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureDir.md).
[`ensureDir` type definition](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/fs-extra/index.d.ts#L47-L50) is not appropriate so I use `ensureDir` for type checking.
I will send PR to DefinitelyTyped.

Reply:
Is this functionally different than? It seems like when we were using mkdirp our options were being ignored. So maybe that was right and we should just be removing the options. Does that seem right?

Reply:
okay nvm, looked into this and it appears correct. Thanks for the call out!